models/event.py

```diff
@@ -145,34 +231,106 @@ class Meta:
     )
     ordering = ('date',)
 
+
 class PostbackURL(models.Model):
   event = models.ForeignKey('Event', on_delete=models.PROTECT, verbose_name='Event', null=False, blank=False, related_name='postbacks')
   url = models.URLField(blank=False,null=False,verbose_name='URL')
   class Meta:
     app_label = 'tracker'
 
+
 class SpeedRunManager(models.Manager):
   def get_by_natural_key(self, name, event):
     return self.get(name=name,event=Event.objects.get_by_natural_key(*event))
 
+
 class SpeedRun(models.Model):
   objects = SpeedRunManager()
   event = models.ForeignKey('Event', on_delete=models.PROTECT, default=LatestEvent)
   name = models.CharField(max_length=64,editable=False)
-  deprecated_runners = models.CharField(max_length=1024,blank=True,verbose_name='*DEPRECATED* Runners') # This field is now deprecated, we should eventually set up a way to migrate the old set-up to use the donor links
+  deprecated_runners = models.CharField(max_length=1024,blank=True,verbose_name='*DEPRECATED* Runners', editable=False) # This field is now deprecated, we should eventually set up a way to migrate the old set-up to use the donor links
   description = models.TextField(max_length=1024,blank=True)
-  starttime = models.DateTimeField(verbose_name='Start Time')
-  endtime = models.DateTimeField(verbose_name='End Time')
-  runners = models.ManyToManyField('Donor', blank=True, null=True)
+  starttime = models.DateTimeField(verbose_name='Start Time', editable=False, null=True)
+  endtime = models.DateTimeField(verbose_name='End Time', editable=False, null=True)
+  order = models.IntegerField(editable=False, null=True)  # can be temporarily null when moving runs around, or null when they haven't been slotted in yet
+  run_time = TimestampField(always_show_h=True)
+  setup_time = TimestampField(always_show_h=True)
```

Comment:
Are you referring to run_time and setup_time? They default to 0 because under the hood they're millisecond time durations that are nicely tarted up when displayed (and parsed back down when input).